streams/src/main/java/org/apache/kafka/streams/processor/internals/StandbyTask.java

```diff
@@ -92,7 +96,7 @@ public StandbyTask(TaskId id,
     public void commit() {
         log.debug("standby-task [{}] Committing its state", id());
         stateMgr.flush(processorContext);
-
+        stateMgr.checkpoint(Collections.<TopicPartition, Long>emptyMap());
```

Comment:
yes it is correct as for standby task there are no ackedOffsets. It uses the restoredOffsets in `ProcessorStateManager`